sir_complainsalot/delphi_sir_complainsalot/check_source.py

```diff
@@ -103,20 +83,37 @@ def check_source(data_source, meta, params, grace, logger):
             logger.info("No signal data retrieved")
             continue
 
-        # convert numpy datetime values to pandas datetimes and then to
-        # datetime.date, so we can work with timedeltas after
         unique_dates = [pd.to_datetime(val).date()
                         for val in latest_data["time_value"].unique()]
-        unique_issues = [pd.to_datetime(val).date()
-                        for val in latest_data["issue"].unique()]
-
         current_lag_in_days = (datetime.now().date() - max(unique_dates)).days
```

Comment:
what happens if unique_dates is empty? `google_symptoms` has no data from the last 14 days.

Reply:
Hmmm good point. Ok, wdyt about this version where we fallback to using metadata (and flag this in the log) if we get no data from the API?

Output using the current prod config is shown below:

{"message": "is 77 days old - (last update: 2020-12-08)", "data_source": "google-symptoms", "signal_and_geo_types": "`ageusia_raw_search: [county, hhs, hrr, msa, nation, state]`\n`ageusia_smoothed_search: [county, hhs, hrr, msa, nation, state]`\n`anosmia_raw_search: [county, hhs, hrr, msa, nation, state]`\n`anosmia_smoothed_search: [county, hhs, hrr, msa, nation, state]`\n`sum_anosmia_ageusia_raw_search: [county, hhs, hrr, msa, nation, state]`\n`sum_anosmia_ageusia_smoothed_search: [county, hhs, hrr, msa, nation, state]`\n", "event": "Signal out of SLA", "logger": "delphi_sir_complainsalot.run", "level": "critical", "timestamp": "2021-02-23T17:59:35.978353Z"}

Reply:
perfect! for signals that haven't updated in two weeks, it won't matter if meta is a day behind